packages/frontend/src/components/DashboardTiles/TileForms/TileModal.tsx

```diff
@@ -88,10 +88,7 @@ export const AddTileModal: FC<AddProps> = ({ onClose, onAddTile, type }) => {
             uuid: uuid4(),
             properties: properties as any,
             type,
-            h: 3,
-            w: 5,
-            x: 0,
-            y: 0,
+            ...getDefaultChartTileSize(undefined), // requires a fetch to know chart type
```

Comment:
It would make more sense to have `DefaultDashboardTileSize` object instead of calling a function about charts when we know it's not a chart. The `getDefaultChartTileSize` would return the  `DefaultDashboardTileSize` by default.

Reply:
Great idea, thanks